src/block/carousel/frontend-carousel.js

```diff
@@ -588,6 +590,37 @@ class _StackableCarousel {
 
 		this.hasTouched = false
 	}
+
+	// Pause autoplay while inline navigation is in progress
+	fixInlineScrollNavigation = () => {
+		let listenerTimeout = null
+		let fallbackTimeout = null
+
+		const handleEndScroll = () => {
+			clearTimeout( fallbackTimeout )
+			this.unpauseAutoplay()
+			document.removeEventListener( 'scroll', scrollListener )
+		}
+
+		// After 100ms of non-scrolling, then the navigation scroll is done, restart the autoplay
+		const scrollListener = throttle( () => {
```

Comment:
I believe you do not need `throttle` here since the scroll handler will just be called rapidly, so you're just re-creating the timeout fast until all the scrolls are finished. You may just need to increase your timeout